src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceDocTransformer.java

```diff
@@ -174,6 +182,29 @@ private TocModuleView generateTocModuleView(
         .build();
   }
 
+  private String getTocDescription(List<String> lines) {
+    StringBuilder builder = new StringBuilder();
+    for (String line : lines) {
+      if (!line.isEmpty()) {
+        int dotIndex = line.indexOf(".");
+        if (dotIndex == line.length() - 1) {
+          builder.append(line);
+          break;
+        }
+
+        if (line.charAt(dotIndex + 1) == ' ') {
+          builder.append(line.substring(0, dotIndex + 1));
+          break;
+        }
+
```

Comment:
Don't you need to find all occurrences of `.` in the line to check if any is followed by whitespace, not just the first one?